code/game/objects/objs.dm

```diff
@@ -10,6 +10,7 @@
 	var/force = 0
 
 	var/list/armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 0, acid = 0)
+	var/hulk_damage = 150 //the damage hulks do on punches to this object, is affected by melee armor

```

Comment:
I am speaking from ignorance here, but could you not override `attack_hulk()` for steampunk structures for reduced damage?